### PR TITLE
Handle singular/plural correctly

### PR DESCRIPTION
**Pull request by ralt**

### CODE REVIEW

File: src/Chat/Plugin/Rebecca.php [hunk 1]
```diff
@@ -40,10 +40,13 @@ private function getCountdown(): string
...
         return sprintf(
-            'Only %d days, %d hours and %d minutes left until Rebeccaday, OMG!',
+            'Only %d %s, %d %s and %d %s left until Rebeccaday, OMG!',
             $timeLeft->days,
+            $timeLeft->days > 1 ? 'days' : 'day',
             $timeLeft->h,
-            $timeLeft->i
+            $timeLeft->h > 1 ? 'hours' : 'hour',
+            $timeLeft->i,
+            $timeLeft->i > 1 ? 'minutes' : 'minute'
```
Comment (PeeHaa) [inline src/Chat/Plugin/Rebecca.php]: > Only 2 days, 1 hour and 0 minute left until Rebeccaday, OMG!

Comment (ralt) [inline src/Chat/Plugin/Rebecca.php]: ?

Comment (ralt) [inline src/Chat/Plugin/Rebecca.php]: Hm, isn't 0 still singular?

Comment (staabm) [inline src/Chat/Plugin/Rebecca.php]: it should be "0 days", "0 hours" or "0 minutes"

**Comment by ralt** [comment]
@PeeHaa PR fixed. English is weird.

File: src/Chat/Plugin/Rebecca.php [hunk 2]
```diff
@@ -40,10 +40,13 @@ private function getCountdown(): string
...
         return sprintf(
-            'Only %d days, %d hours and %d minutes left until Rebeccaday, OMG!',
+            'Only %d %s, %d %s and %d %s left until Rebeccaday, OMG!',
             $timeLeft->days,
+            $timeLeft->days === 1 ? 'days' : 'day',
             $timeLeft->h,
-            $timeLeft->i
+            $timeLeft->h === 1 ? 'hours' : 'hour',
+            $timeLeft->i,
+            $timeLeft->i === 1 ? 'minutes' : 'minute'
```
Comment (PeeHaa) [inline src/Chat/Plugin/Rebecca.php]: Somewhat my bad. But you just copy/pasted it :-)

It will now display 1 days and 2 day :P

**Pull request merged**